### ISSUE TITLE
Relative input/output paths not supported

### ISSUE THREAD

**Issue by ode2code95** [ghzcool/swagger-react-generator#2]
Hello, thanks for providing this time-saving automation.

On Windows 11 with NodeJS v18.18.1, I cannot use relative paths in this script. Absolute paths do work correctly.

I installed the script to my React application folder, and then ran the command given in the readme, with m own relative paths. I tried formatting the relative paths in various ways, `./src/swagger.json`, `/src/swagger.json` (no dot prefix), and `"/src/swagger.json"` (with quotes).

In case this is related, I am using NVM v1.1.9 to manage my Node versions.

**Comment by ghzcool** [comment]
Hi and thank you for you feedback.

I'm unable to reproduce your problem.

Same OS, same NodeJS version. 

node node_modules/swagger-react-generator -I ./src/swagger.json -O ./src/api.js
Generate Swagger React client API
DONE

**Comment by ode2code95** [comment]
Good morning,

I appreciate you taking time to investigate. After copying your example from this post (which incidentally looks identical to the readme) and running it successfully, I am inclined to believe it was a problem with my shell. This morning after a restart it worked correctly. I also believe I was running in PS, which is the default in my Windows Terminal app.

I had just installed the package and did not start a new shell. As mentioned, it worked fine this morning with both PS and Windows cmd.

I added a script to my package.json file to allow me to update the generated API with one command, and that also works like a charm. 👍 

Thanks again for looking into this.
